wizard/WizardModeRemoteNodeWarning.qml

```diff
@@ -0,0 +1,206 @@
+// permitted provided that the following conditions are met:
+// 
+// 1. Redistributions of source code must retain the above copyright notice, this list of
+//    conditions and the following disclaimer.
+// 
+// 2. Redistributions in binary form must reproduce the above copyright notice, this list
+//    of conditions and the following disclaimer in the documentation and/or other
+//    materials provided with the distribution.
+// 
+// 3. Neither the name of the copyright holder nor the names of its contributors may be
+//    used to endorse or promote products derived from this software without specific
+//    prior written permission.
+// 
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY
+// EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF
+// MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL
+// THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+// SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+// PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+// INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+// STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF
+// THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+import QtQuick 2.7
+import QtQuick.Layouts 1.2
+import QtQuick.Controls 2.0
+
+import "../js/Wizard.js" as Wizard
+import "../components" as MoneroComponents
+
+Rectangle {
+    id: wizardModeRemoteNodeWarning
+
+    color: "transparent"
+    property string viewName: "wizardModeRemoteNodeWarning"
+    property bool understood: false
+
+    ColumnLayout {
+        Layout.alignment: Qt.AlignHCenter;
+        width: parent.width - 100
+        Layout.fillWidth: true
+        anchors.horizontalCenter: parent.horizontalCenter;
+
+        spacing: 10 * scaleRatio
+
+        ColumnLayout {
+            Layout.fillWidth: true
+            Layout.maximumWidth: wizardController.wizardSubViewWidth
+            Layout.topMargin: wizardController.wizardSubViewTopMargin
+            Layout.alignment: Qt.AlignHCenter
+            spacing: 0 * scaleRatio
+
+            WizardHeader {
+                title: qsTr("About the simple mode") + translationManager.emptyString
+                subtitle: ""
+            }
+
+            ColumnLayout {
+                spacing: 20 * scaleRatio
+
+                Layout.topMargin: 10 * scaleRatio
+                Layout.fillWidth: true
+
+                Text {
+                    text: qsTr("This mode is ideal for managing small amounts of money. You have access to basic features for making and managing transactions. It will automatically connect to the Monero network so you can start using Monero immediately.") + translationManager.emptyString
```

Comment:
```suggestion
                    text: qsTr("This mode is ideal for managing small amounts of monero. You have access to basic features for making and managing transactions. It will automatically connect to the Monero network so you can start using Monero immediately.") + translationManager.emptyString
```

better IMO :)

Reply:
Oh, good catch.